Makefile

```diff
@@ -55,7 +55,7 @@ LOG_LEVEL := "info"
 endif
 
 ifeq "${DEBUG}" "true"
-LOG_LEVEL := "debug"
+LOG_LEVEL := DEBUG
```

Comment:
This shouldn't be changed. This variable is only used in the run command of `gunicorn` which accepts [the following values](https://docs.gunicorn.org/en/stable/settings.html#loglevel) for `--log-level`, with the default being 'info'
* 'debug'
* 'info'
* 'warning'
* 'error'
* 'critical'